_includes/v21.2/orchestration/start-cockroachdb-secure.md

```diff
@@ -17,6 +17,8 @@
       secretName: cockroachdb.node
     ~~~
 
+1. The StatefulSet configuration deploys CockroachDB into the `default` namespace. To use a different namespace, search for `kind: RoleBinding` and change its `subjects.namespace` to the name of the namespace. Otherwise, a `failed to read secrets` error occurs when you attempt to follow the steps in [Initialize the cluster](#initialize-the-cluster).
```

Comment:
I think a better wording would be one of these:

- change its `subjects.namespace` property 
- change`subjects.namespace`

Reply:
Good call. Done.